docs/source/classroom/cps_ood.md

```diff
@@ -15,19 +15,19 @@ Due to problems with launching OOD on **Safari**, we recommend using **Google Ch
 ## Open JupyterLab For Classes
 
 :::{important}
-The class instructor needs to fill in the: [Discovery Classroom Use Request] You will only be able to find your class resources if a request was already made.
+The class instructor needs to fill in the [HPC Classroom Use Request](https://bit.ly/NURC-Classroom) You will only be able to find your class resources if a request was already made.
 :::
 
-1. In a web browser, go to <http://ood.discovery.neu.edu>. Login with your NU credentials.
+1. In a web browser, go to <http://ood.explorer.northeastern.edu>. Login with your NU credentials.
 
-1. Under the **Courses** menu, select your Class Name (For example: **ALY6080 JupyterLab**).
+2. Under the **Courses** menu, select your Class Name (For example: **ALY6080 JupyterLab**).
```

Comment:
We may want to suggest that they can use the general unless all CPS courses request custom apps.